src/humidity.jl

```diff
@@ -0,0 +1,63 @@
+"""
+Compute the saturation specific humidity for a single atmospheric level.
+"""
+function get_saturation_specific_humidity(
+    T::Array{NF,2},  # Absolute temperature for a single atmospheric level
+    p::Array{NF,2},  # Normalised surface pressure
+    σ::NF,           # σ of the current atmospheric level
+    M::ModelSetup{NF},
+) where {NF<:AbstractFloat}
+    @unpack nlon, nlat = M.geospectral.geometry
+    @unpack e₀, C₁, C₂, T₀, T₁, T₂ = M.params.humidity_coefs
```

Comment:
`M.parameters.abc...`

Reply:
I find it concerning that this slipped through...